README.md

```diff
@@ -69,10 +70,11 @@ http://www.ubuntu.com/legal/contributors
 To get the source and propose a merge, this is what typically needs to
 be done:
 
-     bzr branch lp:snappy my-work
-     cd my-work
-     [hack on mywork]
-     bzr lp-propose
+    cd ~/work/src/github.com/ubuntu-core/snappy
+    git checkout -b my-work
+    [hack on mywork]
+    git push
+    [create pull request]
```

Comment:
This workflow is not actually the one I'd suggest. It won't for people not in the project team, and it will leave the project loaded with small branches.

My suggestion is to encourage people to push their branches to their own space (github.com/chipaca/snappy) and send pull requests from there. This way developers don't need any special permissions to be contributing, and the project remains clean of temporary artifacts. The branches we decide to push into the main project should be those we indeed intend others to see and follow, such as longstanding feature branches that for some reason shall remain unmerged for a longer while.